### ISSUE TITLE
Option to have JIRA ticket marked Resolved on check-in

### ISSUE THREAD

**Issue by bferguso** [BCDevOps/nrdk#47]
It might be nice to add functionality to allow backlog:checkin to resolve the associated JIRA ticket.

**Comment by chobson** [comment]
I don't think that backlog:checkin should resolve the JIRA ticket, as all it does is create a pull request. What would be good is, when the pull request is merged, then at that time the Jira issue is resolved. This will require some exploration of the Jira/BitBucket relationship to determine what is possible, but backlog:checkin should create PRs that resolve Issues if they are able.
